pallets/liquidity-pools/src/message.rs

```diff
@@ -226,7 +226,7 @@ pub enum Message<BatchContent = BatchMessages> {
 		/// The hash of the message which shall be recovered
 		hash: [u8; 32],
 		/// The address of the router
-		address: Address,
+		router: [u8; 20],
```

Comment:
Can anyone please confirm that changing these names is OK?

Reply:
I don't think so. The deserialize part in Solidity would expect 32 bytes 🤔, unless this message has special treatment
cc @hieronx

Reply:
Thanks for explaining the whys! I didn't know the exact reason.

Reply:
OK, I will revert the type, what about the field name then?